apps/minifront/src/components/shared/asset-selector.tsx

```diff
@@ -105,13 +106,17 @@ export const AssetSelector = ({ assets, onChange, value, filter }: AssetSelector
       <DialogContent>
         <div className='flex max-h-screen flex-col'>
           <DialogHeader>Select asset</DialogHeader>
+
           <div className='flex flex-col gap-2 overflow-auto p-4'>
-            <IconInput
-              icon={<MagnifyingGlassIcon className='size-5 text-muted-foreground' />}
-              value={search}
-              onChange={setSearch}
-              placeholder='Search assets...'
-            />
+            <Box spacing='compact'>
+              <IconInput
```

Comment:
I removed the black background and gray border from `<IconInput />`, so I had to wrap all instances of `<IconInput />` in the codebase with `<Box />`